commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/RequestStatus.kt

```diff
@@ -1,6 +1,9 @@
 package ebi.ac.uk.model
 
 enum class RequestStatus {
+    DRAFT,
+    SUBMITTED,
+    ACCEPTED,
```

Comment:
a draft accepted is the same as request requrested is not it?

Reply:
Yes, we don't need this stage anymore